### ISSUE TITLE
[Bug]: i want tot delate my projects but ...

### ISSUE THREAD

**Issue by curokami** [denoland/deploy_feedback#395]
### Problem description

I have created many useless projects due to my operational errors. They are now "pending".
I would like to delete them. But I don't know how to do it. Am I missing something, or is it a bug that I can't erase the pending projects?

### Steps to reproduce

I currently have six pending projects.

### Expected behavior

現在、私はペンディング中のプロジェクトを６つ持っています。

### Environment

The problems I face should not be dependent on my environment.

### Possible solution

In other words... I currently have 6 pending projects.

**Comment by magurotuna** [comment]
Could you elaborate a bit on your issue and how we can reproduce it?

The first thing I would like to clarify is, what do you mean by "pending"? Can you share a procedure to create a pending project? (I assume that we can create a pending project by clicking on "Empty Project" button in https://dash.deno.com/new - correct me if it's wrong)

Next, did you try to do anything in order to delete your projects? I would like to clarify whether 1) you did something to delete them, but that failed for some reason, or 2) you just had no idea, so did nothing.

**Comment by curokami** [comment]
As you surmised, I had mistakenly created several empty projects.
Yesterday, when I accessed the site from my Android phone and tapped on an empty project tile, I was taken to the screen of that project alone, and there was a trash can icon there.
I tapped on it and was able to delete the empty projects.
In other words, I learned that empty projects can be deleted via my phone.

But then why couldn't I erase it on my PC? That question is unanswered.
So today I accessed my deno deploy admin page on my PC and created another empty project. This was an attempt to see if I could delete the project by tapping on a tile (or clicking with the mouse on a PC), which is the same operation as on an Android phone, to call up the project screen and delete the project there.

The result is that clicking on an empty project tile with the mouse on a PC does not change anything.
By the way, the browser on both PC and Android phone is Google Chrome.
Why can't I click on an empty project tile on my PC and call up its own page?
Why the difference in the operating experience between mobile and PC?
I can't find a clue to solve this question of mine.

**Comment by magurotuna** [comment]
Thanks for the clarification - but I still don't get what issue you came across. When I try to simulate what you did on PC, I saw like the following.

https://github.com/denoland/deploy_feedback/assets/23649474/30a4044d-6a54-4b60-ab4f-fa7a6a29b929

Did you see something different? If you could, screenshots or something would be really helpful.

**Comment by curokami** [comment]
I have located the source of the problem.
It seems that one of chrome's extensions, AdBlock, was interfering with the page transitions.

Yesterday I also tested on my pc with firefox, my sub-browser, to see if I could press the project tile.
As a result, I was taken to a single project screen as normal. In other words, I got the intended experience.

No problem on my android phone and firefox. But with chrome the experience is compromised.
What is the difference between the two camps?
When I thought about it, I became very aware of the chrome extensions.

So I removed all chrome extensions.
Bingo! I was able to properly transition to a single project page.

After that, I turned on the extensions one by one to see if they affected the page transition.
As a result, as I said at the beginning, AdBlock was exposed.
Problem solved, thanks for your patience with my crazy feedback. Thanks again.

**Comment by magurotuna** [comment]
Very glad that your issue was solved! I hope you will enjoy the experience with Deno Deploy. I'm going to close this issue as resolved.
